packages/flutter/test/widgets/spell_check_test.dart

```diff
@@ -43,12 +43,12 @@ void main() {
         spellCheckResults,
     );
 
-      expect(textSpanTree, equals(expectedTextSpanTree));
-  });
+    expect(textSpanTree, equals(expectedTextSpanTree));
+    }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.android, TargetPlatform.iOS }));
```

Comment:
Did these tests fail or you just changed them to mobile-only since that's what spell check supports right now?

Reply:
I mean we can have them run on all, but mobile is what matters so I'd figure I'd change it just to be sure we are testing what we support. Your call! I need them to at least run on those too to try the two different ways of building the `TextSpan` tree with spell check results.

Reply:
I say keep it as you have it 👍 .  I've seen unnecessary failures happen because a test was running on a platform that it was irrelevant for.